### PR TITLE
Update permissions.yml yang xiao long

### PR DESCRIPTION
**Pull request by HorlStarAndHarl**

### CODE REVIEW

**Comment by HorlStarAndHarl** [comment]
Proof of all her powers can be found in this video:
https://www.youtube.com/watch?v=QCw_aAS7vWI

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -12441,6 +12441,33 @@ groups:
         - ch.alias.haste1
         - ch.alias.haste2
         - ch.alias.haste3
+ yangxiaolong:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: her name should have 2 spaces in front, not 1

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -12418,7 +12418,7 @@ groups:
         - ch.alias.haste2
         - ch.alias.haste3
         - ch.alias.haste4
-  wolverine:
+   wolverine:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: you gave wolverine 3 spaces, he should also only have 2

File: PermissionsEx/permissions.yml (hunk 1 above)
Comment (HorlStarAndHarl) [reply PermissionsEx/permissions.yml]: Oh hork said only one ok Ill fix it

File: PermissionsEx/permissions.yml (hunk 2 above)
Comment (HorlStarAndHarl) [reply PermissionsEx/permissions.yml]: Oh ok I will fix that too

**Pull request merged**